tests/pages/styleguide.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from base import BasePage
+from page import PageRegion
+
+
+class StyleGuidePage(BasePage):
+
+    _url = '{base_url}/{locale}/styleguide'
+
+    _sidebar_locator = (By.CSS_SELECTOR, '#sidebar nav > ul > .has-children')
+
+    @property
+    def menu(self):
+        return [NavigationMenu(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._sidebar_locator)]
+
+
+class NavigationMenu(PageRegion):
+
+    _link_locator = (By.CLASS_NAME, 'toggle')
+    _menu_locator = (By.CSS_SELECTOR, 'ul > li')
+    _sub_menu_locator = (By.CSS_SELECTOR, 'ul > .has-children')
+
+    @property
+    def sub_menu(self):
+        return [NavigationMenu(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._sub_menu_locator)]
+
+    def show_menu(self):
```

Comment:
Could we rename this to `expand`? I think it makes the test a little more readable.